charmed_openstack_info/data/lp-builder-config/openstack.yaml

```diff
@@ -91,6 +91,15 @@ defaults:
       bases:
         - "22.04"
         - "23.04"
+    stable/2023.2:
+      enabled: True
+      build-channels:
+        charmcraft: "2.2/stable"
```

Comment:
Blocking until https://github.com/canonical/craft-providers/pull/330 gets available in a release, or we agree something different.